ods_ci/tasks/Resources/Provisioning/GPU/gpu_deploy.sh

```diff
@@ -70,6 +70,12 @@ function rerun_accelerator_migration() {
       return 1
   fi
 
+  echo "Waiting for up to 3 minutes until rhods-dashboard deployment is rolled out"
+  oc rollout status deployment.apps/rhods-dashboard -n redhat-ods-applications --watch --timeout 3m
+
+  echo "Verifying that an AcceleratorProfiles resource was created in redhat-ods-applications"
+  oc describe AcceleratorProfiles -n redhat-ods-applications
```

Comment:
Why doing a describe?

Reply:
would it be better to check the instance was created?

Reply:
Describe checks if the resource was created + describes it, for example:
```
$▶ oc describe AcceleratorProfiles -n redhat-ods-applications
Name:         migrated-gpu
Namespace:    redhat-ods-applications
Labels:       <none>
Annotations:  <none>
API Version:  dashboard.opendatahub.io/v1
Kind:         AcceleratorProfile
Metadata:
  Creation Timestamp:  2024-01-09T15:46:11Z
  Generation:          1
  Managed Fields:
    API Version:  dashboard.opendatahub.io/v1
    Fields Type:  FieldsV1
    fieldsV1:
      f:spec:
        .:
        f:displayName:
        f:enabled:
        f:identifier:
        f:tolerations:
    Manager:         unknown
    Operation:       Update
    Time:            2024-01-09T15:46:11Z
  Resource Version:  2416221
  UID:               ce19e4c7-97eb-46b6-b2f4-cc719e0d8e1b
Spec:
  Display Name:  NVIDIA GPU
  Enabled:       true
  Identifier:    nvidia.com/gpu
  Tolerations:
    Effect:    NoSchedule
    Key:       nvidia.com/gpu
    Operator:  Exists
Events:        <none>

```

If the resource is missing it will fail with:
`error: the server doesn't have a resource type "AcceleratorProfiles"`